### DISCUSSION TITLE
Unable to print the FlowTower

### DISCUSSION THREAD

**Discussion by Dustoff74** [5axes/Calibration-Shapes#67] [Q&A]
Hi 5axes!
Thanks for your great stuff, I already used it a lot!

I just can't figure out how to print the Flow Tower successfully. Latest at the 104% cube, the part with the hole disconnects from the one below it. I already tried it three times without luck, using your recommended settings.
Maybe it would be easier if the "cube part" would just be one solid tower? I could still try the pins as they are separate.

**Comment by 5axes** [comment]
Hi I'm sorry to hear that you are having problems with this test.

I think you may already have a problem with the basic setting of your E Steps... Try a simple test using a single cube with no infill and 3 or 4 contours. with a flow at 100%, measure the thickness of the contours to see if you get the desired thickness (be careful to use 3 or 4 contours, tests with a single contour in mode vase  are not a good method for Cura) 

Concerning the distance between the cubes, this characteristic is part of the things we want to test, the ability to detach the cubes is one of the things we want to characterize.

**Reply by Dustoff74** [reply]
Hey 5axes, thanks for your reply.
No need for you to be sorry. I’m pretty sure I messed it up on my side 😉
After printing a calibration cube I think you are right, my e-steps are a little off. 
I’ll adjust and try the tower again!
